ui/src/components/CsvSourceMapping.vue

```diff
@@ -14,6 +14,13 @@
             </div>
           </div>
         </div>
+        <b-message v-if="source.error" type="is-danger" class="content">
+          <p>An error occured while parsing the CSV file:</p>
+          <pre>{{ source.error }}</pre>
+          <p v-if="source.actions.delete">
+            Fix your file, <a @click="deleteSource(source)">delete this one</a> and upload your file again.
```

Comment:
Maybe instead of putting the action link in the message there would actually be a 🗑️ icon like on the mockups there is one showed for tables?

Reply:
There will be both ;)

I want to have a delete button on the source as well, but guiding the user and tell them what to do when there's an error is a great way to improve user experience.